packages/cozy-harvest-lib/src/components/AccountForm/index.jsx

```diff
@@ -47,9 +47,11 @@ export class AccountForm extends PureComponent {
    * @param  {Object} formState See https://github.com/final-form/final-form#formstate
    * @return {Boolean}
    */
+  // Here error is not a validation error but an instance of a
+  // KonnectorJobError, so submitting again is possible
```

Comment:
Nit: I would put this comment above the return line, I am not sure putting the comment here will play well with the JSDoc comment just below.